substrate/frame/nomination-pools/src/lib.rs

```diff
@@ -2714,12 +2749,12 @@ pub mod pallet {
 		/// account).
 		///
 		/// # Conditions for a permissionless dispatch:
-		/// * When pool depositor has less than `MinNominatorBond` staked, otherwise  pool members
+		/// * When pool depositor has less than `MinNominatorBond` staked, otherwise pool members
 		///   are unable to unbond.
 		///
 		/// # Conditions for permissioned dispatch:
-		/// * The caller has a nominator or root role of the pool.
-		/// This directly forward the call to the staking pallet, on behalf of the pool bonded
+		/// * The caller is the pool's nominator or root.
+		/// This directly forwards the call to the staking pallet, on behalf of the pool's bonded
```

Comment:
Similar to `fn nominate` suggestion. Also, the text `pool's bonded` is incorrect I believe, the pool itself is bonded.
```suggestion
         /// This directly forwards the call to an implementation of `StakingInterface` (e.g., `pallet-staking`) through [`T::StakeAdapter`], on behalf of the pool bonded.
```